patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -2548,10 +2575,34 @@
  					aggro -= (int)((1f - stealth) * 1200f);
  					accRunSpeed *= 0.3f;
  					maxRunSpeed *= 0.3f;
-@@ -18147,6 +_,7 @@
+@@ -18094,14 +_,20 @@
+ 			if (manaSick)
+ 				magicDamage *= 1f - manaSickReduction;
+ 
++			/*
+ 			float num32 = meleeSpeed - 1f;
+ 			num32 *= ItemID.Sets.BonusMeleeSpeedMultiplier[inventory[selectedItem].type];
+ 			meleeSpeed = 1f + num32;
++			*/
++			meleeSpeed = new StatModifier(1 + ((meleeSpeed.Additive - 1) * ItemID.Sets.BonusMeleeSpeedMultiplier[inventory[selectedItem].type]), meleeSpeed.Multiplicative);
+ 			if ((double)pickSpeed < 0.3)
+ 				pickSpeed = 0.3f;
+ 
++			// TO-DO: potentially add a setting in Mod that can make this not a thing
+-			if (meleeSpeed > 3f)
++//			if (meleeSpeed > 3f)
```

Comment:
The typical style recommended for commenting out code is as follows:
``\\code``
On the other hand, for comments, just for comparison
``\\ Comment``

This Style is to help tell at a glance whether a comment is code or guidance

Reply:
(Resolved)